dataplane/windows/ipsets/ipsets.go

```diff
@@ -142,12 +143,26 @@ func (s *IPSets) GetIPSetMembers(setID string) []string {
 
 // filterMembers filters out any members which are not of the correct
 // ip family for the IPSet
-func (s *IPSets) filterMembers(members []string) set.Set {
+func (s *IPSets) filterMembers(members []string, setType ipsets.IPSetType) set.Set {
 	filtered := set.New()
 	wantIPV6 := s.IPVersionConfig.Family == IPFamilyV6
+
+	// IPSet members can come in two forms: IP, or IP and port.
+	// To determine the address family for an IP set member, we must first
+	// determine which type of IP set this is.
+	memberIsIPv6 := func(m string) bool {
+		addr := m
+		if setType == ipsets.IPSetTypeHashIPPort {
```

Comment:
Can we just detect `,` and avoid using `ipsets.IPSetTypeHashIPPort`? or Just copy over the definition from `felix/ipsets`?

Reply:
Splitting on "," would probably work, but IMO is unnecessarily fragile - we hit the current bug because we were splitting based on ":", which isn't truly an indicator of whether a member is IPv4 or IPv6. Would rather avoid using string parsing to determine the type of IP set, when we have a `Type` field available to us!